Library/Homebrew/cask/cmd/cat.rb

```diff
@@ -10,7 +10,11 @@ def initialize(*)
 
       def run
         casks.each do |cask|
-          puts File.open(cask.sourcefile_path, &:read)
+          if ENV["BAT_CONFIG_PATH"] = Homebrew::EnvConfig.bat
+            safe_system "#{HOMEBREW_PREFIX}/bin/bat", cask.sourcefile_path
```

Comment:
Should the `brew cask cat` command not reuse the code from `brew cat`?

Reply:
I don't think that's necessary.